src/docfx/build/page/BuildPage.cs

```diff
@@ -93,7 +93,7 @@ public static async Task<IEnumerable<Error>> Build(Context context, Document fil
             }
 
             outputMetadata.Locale = file.Docset.Locale;
-            outputMetadata.TocRel = tocMap.FindTocRelativePath(file);
+            outputMetadata.TocRel = !string.IsNullOrEmpty(inputMetadata.TocRel) ? inputMetadata.TocRel : tocMap.FindTocRelativePath(file);
```

Comment:
So for this type of archived content, it does not need TOC map. So TOC map build can be lazily evaluated.

Reply:
We don't know if all files in archived content has "_tocRel" metadata, but you are right, we can lazy the toc map. I prefer to do it in another PR.

Reply:
yep, just bring it up.